### ISSUE TITLE
When I have trained a model in the PaddleDetection, how to use the trained model to finetune?

### ISSUE THREAD

**Issue by wangchufan** [open-mmlab/mmdetection#2351]
When I have trained a model in the PaddleDetection, how to use the trained model to finetune?

**Comment by hellock** [comment]
We have no experience with Paddle, but you can try converting Paddle checkpoints to the pytorch format.

**Comment by PanJianning** [comment]
1. Get numpy array of the weights from the checkpoint of paddledetection
2. Find the corresponding architecture in mmdet. If there is not a corresponding one, implement it. (eg, paddle often use resnet_vd, but mmdet does not support resnet_vd backbone right now.)
3. Do variable names mapping. PaddleDetection has a diffrenent nameing policy. eg:
```
    'conv1_1_weights': 'conv1.weight',
    'bnv1_1': 'bn1',
    'conv1_2_weights': 'conv2.weight',
    'bnv1_2': 'bn2',
```

**Comment by hellock** [comment]
@PanJianning vd has been supported in https://github.com/open-mmlab/mmdetection/pull/2252, and will be released in v2.0.
